module5/js/script.js

```diff
@@ -112,11 +114,15 @@ function buildAndShowHomeHTML (categories) {
       // being passed into the $dc.loadMenuItems function. Think about what that argument needs
       // to look like. For example, a valid call would look something like this:
       // $dc.loadMenuItems('L')
+
       // Hint: you need to surround the chosen category short name with something before inserting
       // it into the home html snippet.
       //
       // var homeHtmlToInsertIntoMainPage = ....
 
+      //ab: step 3 I'm stuck, I've changed the home-snippet to grab the randomCategoryShortName, but am not sure
+      //where to apply the inserthtml.
```

Comment:
insertHtml is step 4. Here it's asking for something else. Take a look at other usages of `insertProperty` in this file.